dashboard/lib/widgets/user_sign_in.dart

```diff
@@ -56,8 +70,28 @@ class UserSignIn extends StatelessWidget {
                     child: Text(authService.user?.email ?? 'user@example.com'),
                   );
                 }
-                return GoogleUserCircleAvatar(
-                  identity: FirebaseUserIdentity(authService.user!),
+                return CircleAvatar(
+                  foregroundImage: CachedNetworkImageProvider(
+                    authService.user!.photoURL!,
+                  ),
+                  child: Text(
+                    <String?>[
+                          authService.user!.displayName,
+                          authService.user!.email,
+                          '-',
+                        ]
+                        .firstWhere(
+                          (String? str) =>
+                              str != null && str.trimLeft().isNotEmpty,
+                        )!
+                        .trimLeft()
+                        .split(' ')
+                        .map((String str) => str[0])
+                        .join()
+                        .substring(0, 2)
+                        .toUpperCase(),
```

Comment:
It is not immediately clear what you're trying to do here; maybe move this to a helper function?

I think you are trying to get the initials for the name?  Then:

```dart
String getUserInitials(String name) {
  // The RegExp to match the first character of every word
  RegExp regExp = RegExp(r'\b\w');
  final matches = regExp.allMatches(name); 
  return [for (final match in matches) match.group(0)!].join().toUpperCase();
}
```